src/main/scala/com/typesafe/sbt/packager/rpm/Keys.scala

```diff
@@ -60,4 +60,15 @@ trait RpmKeys {
   // Building
   val rpmLint = TaskKey[Unit]("rpm-lint", "Runs rpmlint program against the genreated RPM, if available.")
 
+  val rpmDaemonLogFile = SettingKey[String]("rpm-daemon-log-file", "Name of the log file generated by application daemon")
+
+  val rpmScriptReplacements = SettingKey[Seq[(String, String)]](
```

Comment:
yes, that makes sense @huntc and I like the idea for a general purpose setting `scriptReplacements` and scoping it to the different configurations. This will be easier if there were less settings ;) 

IMHO we should not introduce a new key as removing these is much harder.